### PR TITLE
feat(pie-docs): DSW-1413 add code section for pie-link

### PR DESCRIPTION
**Pull request by raoufswe**
## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/src/components/link/code/code.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+
+## Peer Dependencies
+
+When using `pie-link`, you will also need to include a couple of dependencies to ensure the component renders as expected.
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "See the [PIE Wiki](https://github.com/justeattakeaway/pie-aperture/tree/main) for more information on peer dependencies."
```
Comment (jamieomaguire) [inline apps/pie-docs/src/components/link/code/code.md:141]: this doesn't seem to be a link to the wiki

Comment (raoufswe) [reply apps/pie-docs/src/components/link/code/code.md:141]: nice spot!
I see it's being used on multiple pages and I assume it's copy-paste. 

what do you think the text should be?

Comment (raoufswe) [reply apps/pie-docs/src/components/link/code/code.md:141]: also thinking about it, it isn't peer dependencies per se, we are just linking to another examples repo

**Review by sebastienrocks — APPROVED** [review]

File: apps/pie-docs/src/components/link/code/code.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+For HTML:
+
+```js
+// import as module into a js file e.g. main.js
+import '@justeattakeaway/pie-link'
+```
+
+```html
+<!-- pass js file into <script> tag -->
+<pie-lin href="https://www.pie.design">pie.design</pie-lin>
```
Comment (jamieomaguire) [inline apps/pie-docs/src/components/link/code/code.md:104]:
```suggestion
<pie-link href="https://www.pie.design">pie.design</pie-link>
```

File: apps/pie-docs/src/components/link/code/code.md (hunk 1 above)
Comment (jamieomaguire) [reply apps/pie-docs/src/components/link/code/code.md:141]: I think raise this with @ashleynolan and for now could you please update it to point to the actual wiki or remove this notification? :)

Comment (raoufswe) [reply apps/pie-docs/src/components/link/code/code.md:141]: we don't have a peer dependencies wiki, i wonder if the wording should change?

**Review by jamieomaguire — APPROVED** [review]

File: apps/pie-docs/src/components/link/code/code.md (hunk 1 above)
Comment (raoufswe) [reply apps/pie-docs/src/components/link/code/code.md:141]: ticket number -> DSW-1538

**Pull request merged**